js/package.json

```diff
@@ -6,19 +6,22 @@
   "repository": "git+https://github.com/reteps/dockerfmt/tree/main/js",
   "author": "Peter Stenger <pete@stenger.io>",
   "license": "MIT",
-  "main": "dist/node.js",
-  "types": "dist/node.d.ts",
```

Comment:
I opted to remove these and `browser` as any modern runtime, bundler, or tool will support `exports`. I'm happy to add them back if you think there's a good reason to keep them.

Reply:
LGTM! 😉

Reply:
Then `engines` should be added, otherwise it means it should run in all node versions, which could do not support `exports`.

Reply:
If you want to add a PR for engines @JounQin that would be appreciated. I think it's a good assumption that people will be using node16+ (not sure what is the minimum for `exports`) at this point.

Reply:
https://nodejs.org/api/packages.html#exports

<img width="1030" alt="image" src="https://github.com/user-attachments/assets/659b2f96-af0a-4f96-9327-740faf54acef" />